src/celeritas/em/distribution/WentzelDistribution.hh

```diff
@@ -178,9 +177,10 @@ CELER_FUNCTION real_type WentzelDistribution::operator()(Engine& rng) const
                                       std::sqrt(particle_.beta_sq()));
         real_type xs = mott_xsec(cos_theta)
                        * ipow<2>(this->calculate_form_factor(cos_theta));
-        if (RejectionSampler(xs, helper_.mott_factor())(rng))
+        if (xs < helper_.mott_factor() * generate_canonical(rng))
```

Comment:
I've spent about 2 hours digging into this and I think the actual issue might be that the Mott ratio has substantially different forms between electron and positron. From https://doi.org/10.1016/j.radphyschem.2013.04.020 :

The electron Mott ratios as a function of angle are:
![1-s2 0-S0969806X13002454-gr1_lrg](https://github.com/user-attachments/assets/a159b8d8-f65d-49dc-9992-f946ebd6cc37)

but for positrons they're bounded by 1:
![1-s2 0-S0969806X13002454-gr2_lrg](https://github.com/user-attachments/assets/9e033757-fed5-4f41-8859-f692ee1a7082)

I looked at Lijian et al [1995] and they only describe electrons; the newer reference includes both.

Reply:
Oh that's fantastic, thanks for digging into that and for the reference! Let me see if I can make that update to the Mott ratios first and then revert this change.